ui/AssessmentManager.py

```diff
@@ -31,9 +31,9 @@ def finished_init(self, *args):
         if len(ids) > 2:
             self.user = ids.user
             self.module_number = ids.module_number
-            self.app.title = "Health Friend [Assessment]  ::  " + self.user['first_name'] + " " + self.user['last_name']
-        else:
-            self.app.title = "Health Friend [Assessment]  ::  EWH"
+        #     self.app.title = "Health Friend [Assessment]  ::  " + self.user['first_name'] + " " + self.user['last_name']
+        # else:
+        #     self.app.title = "Health Friend [Assessment]  ::  EWH"
```

Comment:
Can we delete these lines

Reply:
I uncommented it for now, since it only appears on the window's title bar anyways